src/NUnitFramework/tests/Internal/Filters/NotFilterTests.cs

```diff
@@ -67,7 +67,7 @@ public void ExplicitMatchTest()
         {
             var filter = new NotFilter(new CategoryFilter("Dummy"));
 
-            Assert.False(filter.IsExplicitMatch(_topLevelSuite));
+            Assert.True(filter.IsExplicitMatch(_topLevelSuite));
```

Comment:
I'm really not sure about this. Let's discuss more generally what it means for a NotFilter to match explicitly. Maybe it doesn't ever do so.